prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -97,6 +102,11 @@ def __get_bucket_encryption__(self, bucket):
         logger.info("S3 - Get buckets encryption...")
         try:
             regional_client = self.regional_clients[bucket.region]
+        except Exception as error:
+            logger.error(
+                f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+        try:
```

Comment:
Why do you need two `try` blocks?

Reply:
Because the first try does not have the regional_client defined and the second one has it.